fs2/shared/src/main/scala/org/atnos/eff/addon/fs2/AsyncTaskInterpreter.scala

```diff
@@ -17,26 +17,49 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.duration.FiniteDuration
 import scala.util.{Either, Failure, Left, Right, Success}
 
-case class AsyncTaskInterpreter(es: ExecutorServices) extends AsyncInterpreter[Task] { outer =>
+case class AsyncTaskInterpreterEffects()(implicit s: Strategy, sc: Scheduler, es: ExecutorServices) extends AsyncInterpreter[Task] { outer =>
```

Comment:
Do we still need the `ExecutorServices` if we have a `Strategy` and a `Scheduler`?